includes/documents/abstract-wcpdf-order-document.php

```diff
@@ -619,14 +619,19 @@ public function header_logo() {
 	}
 
 	public function get_settings_text( $settings_key, $default = false, $autop = true ) {
-		if ( !empty( $this->settings[$settings_key]['default'] ) ) {
+		if( array_key_exists( 'default', $this->settings[$settings_key] ) ) {
 			$text = wptexturize( trim( $this->settings[$settings_key]['default'] ) );
-			if ($autop === true) {
-				$text = wpautop( $text );
-			}
 		} else {
-			$text = $default;
+			$text = wptexturize( trim( reset( $this->settings[$settings_key] ) ) );
+			if( ! $text ) {
+				$text = $default;
```

Comment:
`$default` should be the fallback for both options (it works like that now, but no longer if you change the first condition, so it's safer as a third `else` condition.